client/signup/steps/plans/index.jsx

```diff
@@ -28,8 +29,10 @@ class PlansStep extends Component {
 			stepSectionName,
 			stepName,
 			goToNextStep,
-			translate
-		} = this.props;
+			translate,
+			signupDependencies: { domainItem }
+		} = this.props,
+			privacyItem = ! isEmpty( cartItem ) && cartItems.domainPrivacyProtection( { domain: domainItem.meta } );
```

Comment:
This breaks if you don't have a domain selected:

`index.jsx:96 Uncaught TypeError: Cannot read property 'meta' of undefined`

To reproduce: Go through Signup and select a free `.wordpress.com` domain, but try to choose a paid plan after that.